packages/components/package.json

```
@@ -14,7 +14,7 @@
     "build": "npm-run-all build:*",
     "build:mitosis": "mitosis build",
     "build:mv-styles": "cpr src build -o -f \"(.ts|.tsx|.md)$\"",
-    "build:sass": "sass build --no-source-map --load-path=node_modules/ --load-path=../../node_modules/ --future-deprecation=import",
+    "build:sass": "sass build --no-source-map --load-path=node_modules/ --load-path=../../node_modules/",
```

```suggestion
    "build:sass": "sass build --no-source-map --load-path=node_modules/ --load-path=../../node_modules/ --future-deprecation=import",
```

why remove it? It might be a good reminder in case somebody still uses it.

---

but than it wouldn't hurt to leave that check in there, would it ?